packages/rlc-common/src/helpers/importsUtil.ts

```diff
@@ -0,0 +1,97 @@
+import { ImportType, Imports } from "../interfaces.js";
+
+/**
+ * Build the common imports for generated SDK
+ * @param branded whether to use azure-branded imports, the default value is true for azure branded
+ * @returns
+ */
+export function build3ndPartyImports(branded = true): Imports {
```

Comment:
nit: I would prefer to call this `buildThirdPartyImports`

Reply:
Actually can we just call this buildCoreImports or buildRuntimeImports, I feel like 3rd party is a bit misleading. I would expect to see things external to Microsoft in here like lodash or axios when reading 3rd party

Reply:
Got it and update to buildRuntimeImports